csfieldguide/templates/interactives/matrix-simplifier.html

```diff
@@ -0,0 +1,173 @@
+{% extends interactive_mode_template %}
```

Comment:
Is the lack of background colour normal in the screenshot icons? (this is from the project panel 3D transforms)
It's also missing the matrices themselves but I guess that's a bit tricky
![image](https://user-images.githubusercontent.com/25916475/64213744-1292b700-cf02-11e9-812e-d7518593f4a8.png)

Reply:
Looking more closely this is pretty normal for our interactives - just most of the more obvious ones never turn up on different coloured panels